src/Compilers/CSharp/Test/Emit2/Diagnostics/DiagnosticSuppressorTests.cs

```diff
@@ -70,6 +70,72 @@ void M(int i)
             VerifySuppressedAndFilteredDiagnostics(compilation, analyzers);
         }
 
+        [WorkItem(62540, "https://github.com/dotnet/roslyn/issues/62540")]
+        [Fact]
+        public void TestSuppression_CompilerSyntaxBindingError_AndSuppressibleWarning()
+        {
+            const string SourceCode = @"
+                public class MyClass
+                {
+                    void MyPrivateMethod(int i)
+                    {
+                        // warning CS1522: Empty switch block
+                        // NOTE: Empty switch block warning is reported by the C# language parser
+                        switch (i)
+                        {
+                        }
+                    }
+                }
+                public class YourClass
+                { 
+                    void YourPrivateMethod()
+                    {
+                        // Cannot access private method
+                        new MyClass().MyPrivateMethod();
+                    }
+                }";
+
+            var compilation = CreateCompilation(SourceCode);
```

Comment:
Please include `compilation.VerifyDiagnostics()` in the test like we do in `TestSuppression_CompilerSyntaxWarning`.

Reply:
Added: https://github.com/dotnet/roslyn/pull/62641/files#diff-b4820c44d14e9bd751949497072f4fbbe861bf55b549b50fd126d05c3bb0c7dcR98

Reply:
Added: https://github.com/dotnet/roslyn/pull/62641/files#diff-b4820c44d14e9bd751949497072f4fbbe861bf55b549b50fd126d05c3bb0c7dcR98